docs/operator-manual/high_availability.md

```diff
@@ -207,6 +207,9 @@ If the manifest generation has no side effects then requests are processed in pa
 Argo CD aggressively caches generated manifests and uses the repository commit SHA as a cache key. A new commit to the Git repository invalidates the cache for all applications configured in the repository.
 This can negatively affect repositories with multiple applications. You can use [webhooks](https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/webhook.md) and the `argocd.argoproj.io/manifest-generate-paths` Application CRD annotation to solve this problem and improve performance.
 
+Note: While the documentation mentions using webhooks with the `argocd.argoproj.io/manifest-generate-paths` annotation, starting with 
+Argo CD v2.11 this annotation can also be used **without configuring any webhooks**. Webhooks are not a pre-condition for this feature. You can rely on the annotation alone to optimize manifest generation.
+
```

Comment:
I think this generally LGTM, although perhaps this can be made clearer in this note that this applies for all apps since 2.11. Something like 

```suggestion
Note: This annotations is available to use for webhooks, and since Argo CD v2.11 this annotation can also be used **without needing to configure any webhooks**. Webhooks are not a pre-condition for this feature. You can rely on the annotation alone to optimize manifest generation.
```

? And then remove the mention of webhooks above